applications/EigenSolversApplication/custom_solvers/eigen_direct_solver.h

```diff
@@ -23,6 +23,7 @@
 #include "includes/define.h"
 #include "includes/ublas_interface.h"
```

Comment:
do you still need this if you have the wrapper?

Reply:
not because of the wrapper...